core/src/main/java/org/elasticsearch/search/suggest/SuggestBuilder.java

```diff
@@ -98,6 +98,15 @@ public SuggestBuilder addSuggestion(SuggestionBuilder<?> suggestion) {
         return suggestions;
     }
 
+    /**
+     * Returns the possibly null global suggest text that
+     * should be applied as the text for all suggesters.
+     */
+    @Nullable
+    public String getGlobalText() {
```

Comment:
I'd say `globalText` is better here, because it makes the intent much clearer. I don't think we need to reflect the json parameter name in the Java API, but I might be wrong there.